crates/uv/src/commands/python/install.rs

```diff
@@ -312,6 +312,7 @@ pub(crate) async fn install(
         installation.ensure_externally_managed()?;
         installation.ensure_sysconfig_patched()?;
         installation.ensure_canonical_executables()?;
+        installation.ensure_dylib_patched()?;
```

Comment:
As you mentioned, this probably shouldn't be a hard error to start. We can turn it into a warning here, or in the called function. Here seems a bit more ergonomic because in a future change, we could push it to a `warnings` list like we do for `errors`.

Reply:
I've done this in [95d3adf](https://github.com/astral-sh/uv/pull/10629/commits/95d3adf851f75b53b5b52f35aee27e5e685dbe8a)
There's a bit of duplication with respect to the warning message, but since it's just two callsite it's probably acceptable. Happy to move it into the function itself if you prefer.